pkg/types/installconfig.go

```diff
@@ -42,6 +42,13 @@ var (
 		baremetal.Name,
 		none.Name,
 	}
+	// NetworkPluginNames is a slice of all available network plugins
+	// supported by the installer out-of-box. This is the list of
+	// network plugins presented to the user in the interactive wizard.
+	NetworkPluginNames = []string{
+		"OVNKubernetes",
+		"OpenShiftSDN",
```

Comment:
we should add "Kuryr" here too

Reply:
Sure, I can add Kuryr. I think first we need to settle upon if we need to allow configuring network plugins via the installer.